app/views/posts/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Post details pages</h1>
\ No newline at end of file
+<h1>Post details pages</h1>
+<div class="post_details">
+  <div class="post_header">
+    <p><%= @post.title %> by <%= @post.author.name%></p>
+    <p>comment: <%= @post.comments.count %>, likes: <%= @post.likes.count %></p>
+  </div>
+  <div class="post_content">
+    <p>
+     <%= @post.text %>
+    </p>  
+  </div>
+  <div>
+    <% if @post.most_recent_comments.empty? %>
+        <p>No comment for this post</p>
+    <% else %>
+      <% @post.most_recent_comments.each do |comment| %>
+        <%= comment.author.name %>  :<%= comment.text %>
+      <% end %>
+    <% end %>
+  </div>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/169436670-7a60cbeb-6fda-42c2-90ed-3188e9245605.png)


- [ ] Kindly note that you need to follow the wireframes that were provided in the requirements, I know that the style does not matter but you need to add some basic CSS to follow the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md). check the wireframes :point_down: 

![image](https://user-images.githubusercontent.com/65068771/169435586-73bdb804-e322-44fe-a337-6f7ca8c310d0.png)